security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -8,24 +8,48 @@ toc::[]
 
 == Purpose
 
-Machine Config Operator certificates are used to secure connections between the Red Hat Enterprise Linux CoreOS (RHCOS) nodes and the Machine Config Server.
+This certificate authority is used to secure connections from nodes to Machine Config Server (MCS) during initial provisioning.
+
+There are two certificates:
+. A self-signed CA, the MCS CA
+. A derived certificate, the MCS cert
+
+=== Provisioning details
+
+{product-title} installations using {op-system-first} are installed using Ignition. This process is split into two parts:
```

Comment:
```suggestion
{product-title} installations that use {op-system-first} are installed by using Ignition. This process is split into two parts:
```
From IBM Style Word Usage:
>using _noun_
To avoid ambiguity, replace this gerund with either _by using_ or _that uses_.